### ISSUE TITLE
1D noise?

### ISSUE THREAD

**Issue by hiIExistMan** [josephg/noisejs#29]
iv'e search for 1D but only found 2 & 3D could you please expand the library to support 1D?

**Comment by bionboy** [comment]
Last commit was 2017 just so you know

**Comment by josephg** [comment]
Yeah I'm probably not adding 1d noise. Feel free to fork! You can get 1d noise by using the 2d noise functions and setting y=1 or something.

**Comment by bjazdzyk** [comment]
@josephg 
I've added pull request about it
`https://github.com/josephg/noisejs/pull/34`
